lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -381,6 +385,21 @@ data ByronWalletPostData mw = ByronWalletPostData
     , passphrase :: !(ApiT (Passphrase "encryption"))
     } deriving (Eq, Generic, Show)
 
+-- | A root private key, encrypted using a given passphrase.
+-- The underlying key should contain: private key, chain code, and public key
+newtype EncryptedRootXPrv =
+    EncryptedRootXPrv { getEncryptedRootXPrv :: ByteString }
+    deriving (Generic, Eq, Show)
+
+encryptedRootXPrvBytesLength :: Int
+encryptedRootXPrvBytesLength = 128
+
+data ByronWalletFromXPrvPostData = ByronWalletFromXPrvPostData
+    { name :: !(ApiT WalletName)
+    , encryptedRootPrivateKey :: !EncryptedRootXPrv
+    , passphraseHash :: !(ApiT (Hash "Password"))
```

Comment:
It should be `Hash "encryption"` to match other similar data-types.

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1439/commits/2b5bfa6131d0000cef287b2d5dd531c5692b40d9